tasks/utilities/data_handling/task_gather_scatter.wdl

```diff
@@ -0,0 +1,189 @@
+version 1.0
+
+task gather_scatter {
+  input {
+    String samplename
+    File? taxon_ids
+    # krakentools outputs
+    File? organism
+    File? extracted_read1
+    File? extracted_read2
+    File? krakentools_docker
+    # fastq_scan outputs
+    File? fastq_scan_num_reads_binned1
+    File? fastq_scan_num_reads_binned2
+    File? fastq_scan_num_reads_binned_pairs
+    File? fastq_scan_docker
+    File? fastq_scan_version
+    # Assembly 
+    File? metaspades_warning
+    File? pilon_warning
+    File? assembly_fasta
+    # quast outputs
+    File? quast_genome_length
+    File? quast_number_contigs
+    File? quast_n50
+    File? quast_gc_percent
+    # consensus qc outputs
+    File? number_N
+    File? number_ATCG
+    File? number_Degenerate
+    File? number_Total
+    File? percent_reference_coverage
+    # pangolin outputs
+    File? pango_lineage
+    File? pango_lineage_expanded
+    File? pangolin_conflicts
+    File? pangolin_notes
+    File? pangolin_assignment_version
+    File? pangolin_versions
+    File? pangolin_docker
+    # Nextclade outputs for non-flu
+    File? nextclade_version
+    File? nextclade_docker
+    File? nextclade_ds_tag
+    File? nextclade_aa_subs
+    File? nextclade_aa_dels
+    File? nextclade_clade
+    File? nextclade_lineage
+    File? nextclade_qc
+    # Nextclade outputs for flu HA
+    File? nextclade_ds_tag_flu_ha
+    File? nextclade_aa_subs_flu_ha
+    File? nextclade_aa_dels_flu_ha
+    File? nextclade_clade_flu_ha
+    File? nextclade_qc_flu_ha
+    # Nextclade outputs for flu NA
+    File? nextclade_ds_tag_flu_na
+    File? nextclade_aa_subs_flu_na
+    File? nextclade_aa_dels_flu_na
+    File? nextclade_clade_flu_na
+    File? nextclade_qc_flu_na
+    # IRMA outputs
+    File? irma_version
+    File? irma_docker
+    File? irma_type
+    File? irma_subtype
+    File? irma_subtype_notes
+    # GenoFLU outputs
+    File? genoflu_version
+    File? genoflu_genotype
+    File? genoflu_all_segments
+    # abricate outputs
+    File? abricate_flu_type
+    File? abricate_flu_subtype
+    File? abricate_flu_database
+    File? abricate_flu_version
+
+    # runtime parameters
+    String docker = "us-docker.pkg.dev/general-theiagen/theiagen/terra-tools:2023-03-16"
+    Int disk_size = 50
+    Int cpu = 2
+    Int memory = 8
+  }
+  command <<<
+    python3<<CODE
+    import pandas as pd
+    import numpy as np
+    import os
+    import json
+
+    def load_json_data(file_path, column_name, df):
+      if os.path.exists(file_path):
+        with open(file_path, 'r') as file:
+          json_data_from_file = json.load(file)
+          df_from_file = pd.DataFrame(json_data_from_file, columns=[column_name])
```

Comment:
I think in this case we didn't add a try/except since the input files are generated by a standard built-in WDL function. Empty files shouldn't exist based on the built-in function behavior, and I don't except any malformed JSONs since the WDL version is constant. I don't anticipate there being issues, but we can add a block if you think it's necessary.